entrypoint.sh

```diff
@@ -42,7 +40,14 @@ fi
 
 ### GEOUPDATEIP
 echo "LAUNCH GEOIP UPDATE"
-/usr/bin/geoipupdate --verbose --output --database-directory="${GEOIPUPDATE_DB_DIR}"
+if [ "${GEOIPUPDATE_DRYRUN-default}" == "default" ]; then #if GEOIPUPDATE_DRYRUN is not set
+    /usr/bin/geoipupdate --verbose --output --database-directory="${GEOIPUPDATE_DB_DIR}"
```

Comment:
```suggestion
    geoipupdate --verbose --output --database-directory="${GEOIPUPDATE_DB_DIR}"
```

Let the $PATH do its magic ;)